content/how-to/lazy-load/react.md

```diff
@@ -0,0 +1,151 @@
+---
+title: How to lazy load React 
+contributors:
+  - iammerrick
+---
+
+A component can lazily load dependencies without it's consumer knowing using higher order functions, or a consumer can lazily load it's children without its children knowing using a component that takes a function and collection of modules, or some combination of both.
```

Comment:
I'll link to Higher Order Components on React patterns... http://reactpatterns.com/#Higher-order component